pkg/operator/ceph/cluster/controller.go

```diff
@@ -780,6 +782,53 @@ func (c *ClusterController) onDeviceCMUpdate(oldObj, newObj interface{}) {
 // Delete event functions
 // ************************************************************************************************
 
+func (c *ClusterController) dataDirHostPathCleanUp(clust *cephv1.CephCluster) {
+	logger.Infof("starting data dir host path clean up for cluster - %+v", clust)
+	command := []string{"rm", "rf", clust.Spec.DataDirHostPath}
```

Comment:
Please expend the args to their full meaning, as well as adding `--verbose`

Reply:
What if we have multiple clusters running with the same data dir host path?
I think you need to delete the namespace folder only.

Reply:
Additionally, without the `-` in the second argument, wouldn't this do nothing?

Reply:
@BlaineEXE Fixed it. 

@leseb For deleting only the namespace directory : 

The `MON` directory is placed outside the namespace folder in the Host. 
`[vagrant@worker0 rook]$ ls
mon-a  rook-ceph 
`
So `mon-a` folder still remains after deleting the namespace folder. 
`
[vagrant@worker0 rook]$ ls
mon-a
`
So would this affect the cleanup process ?